llzk-sys/build-support/src/llzk.rs

```diff
@@ -0,0 +1,78 @@
+use anyhow::{Context, Result};
+use bindgen::Builder;
+use cc::Build;
+use std::path::{Path, PathBuf};
+
+use super::config::{bindgen::BindgenConfig, cc::CCConfig, cmake::CMakeConfig};
+
+pub const LIBDIR: &'static str = "lib";
+
+pub struct LlzkBuild<'a> {
+    path: PathBuf,
```

Comment:
```suggestion
    dst_path: PathBuf,
```
Because that's the destination of the build, right?